.asf.yaml

```diff
@@ -18,18 +18,19 @@ github:
   description: "Mirror of Apache ActiveMQ Artemis"
   homepage: https://activemq.apache.org/components/artemis
   labels:
-    - Apache
-    - ActiveMQ
+    - apache
+    - activemq
+    - activemq-artemis
```

Comment:
I checked that there are repos with a minus in the label, https://github.com/topics/activemq-artemis